src/readfish/__about__.py

```diff
@@ -3,3 +3,8 @@
 """
 
 __version__ = "2024.0.0"
+# Warning is the last supported version of MinKNOW for this version of readfish
```

Comment:
I think this is duplicated from `__compatability__.py`

Reply:
Resolved.